cmd/main.go

```diff
@@ -47,7 +47,7 @@ func main() {
 	repositories := bootstrap.PrepareRepositories(db)
 
 	// Prepare Services
-	services = bootstrap.PrepareServices(configuration, parsedBridge.Networks, *clients, *repositories)
+	services = bootstrap.PrepareServices(configuration, parsedBridge.Networks, *clients, *repositories, parsedBridge)
```

Comment:
Can we have only `parseBrdige` as a function parameter and remove the `parsedBridge.Networks`?

Reply:
I don't think it's a good idea to pass the whole parseBridge since we only use networks from it.